src/types.ts

```diff
@@ -169,6 +169,13 @@ export interface Options {
    */
   markdownItSetup?: (MarkdownIt: MarkdownItAsync) => void | Promise<void>
 
+  /**
+   * Wrap the html in a div
+   *
+   * @default true
```

Comment:
The default value doesn't seem to be assigned anywhere

Reply:
Thanks for the the feedback. I've added a default and quickly tested that it functions as expected locally.

(Previously this PR did unintentionally change the default behaviour.)